wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/WireTextField.swift

```diff
@@ -96,18 +97,60 @@ class WireTextField: UITextField {
     }
 
     private func setupTextFieldEvents() {
-        addTarget(self, action: #selector(textFieldDidStartEditing), for: .editingDidBegin)
-        addTarget(self, action: #selector(textFieldDidFinishEditing), for: .editingDidEnd)
```

Comment:
question: why do we replace the target/action method by action closures?

Reply:
We're trying to get rid of the #selector pattern, that's why we're removing it and replacing it with more modern APIs